runtime/auth/aws-signing-crt/jvm/test/aws/smithy/kotlin/runtime/auth/awssigning/crt/CrtMiddlewareSigningTest.kt

```diff
@@ -1,12 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0
- */
-package aws.smithy.kotlin.runtime.auth.awssigning.crt
-
-import aws.smithy.kotlin.runtime.auth.awssigning.AwsSigner
-import aws.smithy.kotlin.runtime.auth.awssigning.tests.MiddlewareSigningTestBase
-
-class CrtMiddlewareSigningTest : MiddlewareSigningTestBase() {
-    override val signer: AwsSigner = CrtAwsSigner
-}
```

Comment:
**Question**: We no longer test the CRT signing middleware?

Reply:
Relocated to `http-auth-aws`